src/main/routing/router_queue_codel.c

```diff
@@ -107,8 +107,7 @@ static void _routerqueuecodel_free(QueueManagerCoDel* queueManager) {
 }
 
 static inline guint64 _routerqueuecodel_getPacketLength(Packet* packet) {
-    return (guint64)(packet_getPayloadLength(packet) + packet_getHeaderSize(packet) +
-                     CONFIG_HEADER_SIZE_ETH);
+    return (guint64)(packet_getPayloadLength(packet) + packet_getHeaderSize(packet));
```

Comment:
Ah, OK, you did define a convenience function here. I think a better place for this to live is `src/main/routing/packet.h`, so that other callers can use this too.